docs/what-is-tizen/versions/tizen-4-0-m1.md

```diff
@@ -1,64 +1,60 @@
 
+Additionally, Tizen Open Source Project worked closely with Microsoft to develop [Visual Studio Tools for Tizen](https://news.samsung.com/global/samsungs-third-tizen-net-developer-preview-introduces-new-visual-studio-tools), which are integrated into Visual Studio, a popular IDE (Integrated Development Environment), and Xamarin.Forms, a cross-platform UI toolkit. Because Xamarin.Forms supports cross platforms, you can build native UIs for Android, iOS, and Windows from a single, shared C# code base, and easily port these applications to Tizen.
 
-## **A New Development Experience for a Strengthened Application Ecosystem**
-
-While many existing OS platforms are limiting due to their framework, policies or business models, Tizen 4.0 aims to promote innovation and the expansion of the application ecosystem by giving developers more freedom and flexibility.
-
-One way the platform accomplishes this is through the introduction of .NET. With .NET, Tizen developers can enjoy C#, the familiar, industry-leading programming language and one of the richest standard libraries.
+Since announcing its collaboration with Microsoft on .NET open-source projects in November 2016, Tizen has steadily released preview versions of the Tizen .NET SDK to help you build more powerful applications and to encourage your participation in the ongoing development of the Tizen application ecosystem.
 
-Additionally, Tizen Open Source Project worked closely with Microsoft to develop [Visual Studio Tools for Tizen](https://news.samsung.com/global/samsungs-third-tizen-net-developer-preview-introduces-new-visual-studio-tools), which are integrated into Visual Studio, a popular IDE (Integrated Development Environment), and Xamarin.Forms, a cross-platform UI toolkit. Because Xamarin.Forms supports cross platforms, developers can build native UIs for Android, iOS and Windows from a single, shared C# codebase, and easily port these apps to Tizen.
+**Figure: Project Wizard in Visual Studio Tools for Tizen**
 
-Since announcing its collaboration with Microsoft on .NET open-source projects last November, Tizen has steadily released preview versions of the Tizen.NET SDK to help developers build more powerful apps and to encourage their participation in the ongoing development of the Tizen application ecosystem.
+![Project Wizard in Visual Studio Tools for Tizen](media/vstools4tizen_project_wizard.png)
 
-![Visual Studio Tools for Tizen > Project Wizard](media/vstools4tizen_project_wizard.png)
+Tizen .NET currently supports Microsoft's .NET Standard 2.0, and provides a stable Tizen C# development environment with the release of the Visual Studio Tools for Tizen v1.0 this fall.
 
+The preview versions of Tizen.NET have garnered the attention of developers across the world, including those who attended the Microsoft Build 2017 held in Seattle, Microsoft Build Tour in Seoul, and Tizen App Development Seminar at Konkuk University in South Korea earlier this month.
```

Comment:
Which month?

Reply:
Please delete the above sentence. 

* The preview versions of Tizen.NET have garnered the attention of developers across the world, including those who attended the Microsoft Build 2017 held in Seattle, Microsoft Build Tour in Seoul, and Tizen App Development Seminar at Konkuk University in South Korea earlier this month.*

Reply:
OK